index.js

```diff
@@ -46,11 +49,30 @@ const urlMap = {};
 
 const rootValue = {
     shortenUrl: ({ url }) => {
+
+        if (!validUrl.isUri(url)) {
+            throw new Error("This URL is Invalid.");
```

Comment:
@OlusanyaJegs Have you tested that this doesn't work with urls that don't exist but have a write format?

Reply:
I have tested it but the condition wasn't working as it should. It validates any url that has "https" and invalidates without it.

Same validation issue i was having when i used the "url-exists" npm package